controllers/articles.controllers.js

```diff
@@ -0,0 +1,20 @@
+const { fetchArticle } = require('../models/articles.models')
+
+function getArticle(req, res, next) {
+    const { article_id } = req.params;
+
+    fetchArticle(article_id)
+    .then((article) => {
+        
+        if(!article) {
+            return res.status(404).send({ msg : 'we dont find yor article' })
+        }
+
```

Comment:
I would make the case that this error handling logic should be in the model function ( instead of the controller )
and that the controller should use the `.catch(next)` to send it back to be handled by error handling middleware
[see notes for more info](https://notes.northcoders.com/courses/js-back-end/error-handling#custom-errors)